websites/S/Smash Pros/metadata.json

```
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "596877893137596416",
+		"name": "c_fornow"
+	},
+	"service": "Smash Pros",
+	"description": {
+		"en": "Smash Pros is a skill based matchmaking service for Super Smash Bros. Ultimate"
+	},
+	"url": "smashpros.gg",
```

Website supports www subdomain
```suggestion
	"url": [
		"smashpros.gg",
		"www.smashpros.gg"
	],
```